omod/src/main/java/org/openmrs/module/radiologyapp/fragment/controller/RadiationResultsFragmentController.java

```diff
@@ -112,7 +112,7 @@ public SimpleObject saveXrayResults(UiUtils uiUtils,
                         File f = new File(imgDir, file.getOriginalFilename());
                         Files.copy(stream, f.toPath());
 
-                        Concept imageConcept = Context.getConceptService().getConceptByUuid("f53f4215-a17b-4516-b33f-854ffe663f61");
+                        Concept imageConcept = Context.getConceptService().getConceptByUuid("bb6055a3-7310-42a3-a752-8d5f1a5d4ee1"); //100126232 on afyaehms
```

Comment:
You created this concept in your local machine or it is an existing one

Reply:
Anything I will need to do to make it work in my machine?

Reply:
create it on your local, just copy as-is from the afya server. or we can have @Bushisky create it on the next conceptsdb  dump

Reply:
> You created this concept in your local machine or it is an existing one

just created it as a holder concept on my local to be able to test the module. You can as well create then override it later when we load the DB bush will provide.